scipy/special/tests/test_basic.py

```diff
@@ -3416,5 +3416,412 @@ def xfunc(delta, r):
     assert_func_equal(special.pseudo_huber, w, z, rtol=1e-13, atol=1e-13)
 
 
+owenst_dt = [[(1.9508080482482910156250000000000000000000000000000000000000000000000000000000000000000000000000000000e+00), (9.7540378570556640625000000000000000000000000000000000000000000000000000000000000000000000000000000000e-02), (2.2942365980155351117754771526378292708740231264960539316842063696200884829628237039895484898529192823e-03)],
```

Comment:
If this is the test data from boost, it should be put in a file under `data/boost/owenst_ipp` and added to the TESTS list in `test_data.py:test_boost`. (Also if it's some other test data, long data set like this should be put in a data file.)

After adding the text data, the npz data file needs to be regenerated using utils/makenpz.py